src/cmd/bootstrap.ts

```diff
@@ -0,0 +1,141 @@
+import { copyFileSync, existsSync, mkdirSync, readdirSync, rmSync, writeFileSync } from 'fs'
+import { copy } from 'fs-extra'
+import { copyFile } from 'fs/promises'
+import { Argv } from 'yargs'
+import { $ } from 'zx'
+import { OtomiDebugger, terminal } from '../common/debug'
+import { env } from '../common/envalid'
+import { BasicArguments, loadYaml, setParsedArgs } from '../common/no-deps'
+import { cleanupHandler, otomi } from '../common/setup'
+import { genSops } from './gen-sops'
+
+const fileName = 'bootstrap'
+let debug: OtomiDebugger
+
+export type Arguments = BasicArguments
+
+/* eslint-disable no-useless-return */
+const cleanup = (argv: Arguments): void => {
+  if (argv.skipCleanup) return
+}
+/* eslint-enable no-useless-return */
+
+const setup = (argv: Arguments): void => {
+  if (argv._[0] === fileName) cleanupHandler(() => cleanup(argv))
+  debug = terminal(fileName)
+}
+
+const rollBack = (): void => {
+  const dirContent = readdirSync(env.ENV_DIR)
+  dirContent.map((item) => rmSync(item, { recursive: true, force: true }))
+}
+
+const generateLooseSchema = (currDir: string) => {
+  const schemaPath = '.vscode/values-schema.yaml'
+  const targetPath = `${env.ENV_DIR}/${schemaPath}`
+  const sourcePath = `${currDir}/values-schema.yaml`
+
+  const valuesSchema = loadYaml(sourcePath)
+  const trimmedVS = JSON.stringify(valuesSchema, (k, v) => (k === 'required' ? undefined : v), 2)
```

Comment:
Instead of `undefined` it is better to use [].
> The required keyword takes an array of zero or more strings. Each of these strings must be unique.
See: https://json-schema.org/understanding-json-schema/reference/object.html#required-properties

Reply:
This function should remove any value that is required - generateLooseSchema in the old CLI does the same thing: https://github.com/redkubes/otomi-core/blob/master/bin/bootstrap.sh#L13

Please advice why it should become an empty array then?